src/Components/Header.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import style from './style/Header.module.css';
+import logo from '../Assets/Images/planet2.png';
+
+const HeaderStyle = {
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  padding: '1rem 5rem',
+  backgroundColor: '#fff',
+  boxShadow: ' 5px 5px 5px #ddd',
+};
+
+const activeStyle = {
+  color: 'darkcyan',
+  borderBottom: '2px solid',
+  paddingBottom: '.2rem',
+};
+
+const separator = {
+  border: '1px solid black',
+  height: '15px',
+};
+
+export default function Header() {
+  return (
+    <>
+      <header style={HeaderStyle} className={style.header}>
+        <NavLink to="/">
+          <div className="logo">
+            <img src={logo} alt="" width={50} height={50} />
+          </div>
+        </NavLink>
+        <nav className={style.navigation}>
+          <NavLink to="/" style={({ isActive }) => (isActive ? activeStyle : undefined)}> Rocket </NavLink>
+          <NavLink to="/dragon" style={({ isActive }) => (isActive ? activeStyle : undefined)}> Dragons </NavLink>
+          <NavLink to="/missions" style={({ isActive }) => (isActive ? activeStyle : undefined)}> Missions </NavLink>
+          <div style={separator} />
+          <NavLink to="/myProfile" style={({ isActive }) => (isActive ? activeStyle : undefined)}> My Profile </NavLink>
+        </nav>
+      </header>
+    </>
+  );
+}
```

Comment:
- [ ] Only the active menu item should be underlined. Right now the rockets item is always underlined irrespective of which page I am on. As you can see I am on the Missions page, but the rockets item is still underlined. Kindly fix that.

  ![shd](https://user-images.githubusercontent.com/36542986/194512082-c3152480-22c9-4b95-9d8e-0de0bbcbf60f.jpg)